### ISSUE TITLE
Import option?

### ISSUE THREAD

**Issue by jamesxv7** [erming/resume-editor#17]
I don't if it is me, but I dont see any import option in the editor.

**Comment by abbondanza** [comment]
I was just about to post the same issue. 

Even when pasting the JSON, the UI does not update. 

Having a two-way data bind would help here, for both issues. (I noticed there IS two-way data binding, but this is only for fields that are non-objects.)

**Comment by abbondanza** [comment]
You can always paste the JSON

**Comment by brylie** [comment]
I don't see import either. This would be a great feature, so that I can update my resume periodically.

**Comment by Svish** [comment]
Yeah... *definitely* needs an "Upload JSON" next to the "Download JSON"...
